as_perimenoun_oi_gynaikes.ass

```diff
@@ -556,47 +557,47 @@ Dialogue: 0,0:41:08.99,0:41:10.76,Default,,0,0,0,,Is it the refs fault \Nif you
 Dialogue: 0,0:41:10.76,0:41:13.56,Default,,0,0,0,,- Was there a score reversal at the last minute?\N- Three meters from the penalty box?!
 Dialogue: 0,0:41:13.56,0:41:15.99,Default,,0,0,0,,Are you kidding me?\N Manolas tackles Skartados...
 Dialogue: 0,0:41:15.99,0:41:18.64,Default,,0,0,0,,...sends him airborne for three meters...
-Dialogue: 0,0:41:18.64,0:41:21.09,Default,,0,0,0,,...and falls in the penatly box!\N - He was faking it!
+Dialogue: 0,0:41:18.64,0:41:21.09,Default,,0,0,0,,...and falls in the penatly box!\N - Was he a bird?!

```

Comment:
"Being a bird" makes no sense in English AFAIK.

Reply:
In greek "Πουλί ήτανε;" also it doesn't make more sense. There is no expression.
"Πουλί ήτανε;" is a ironic reference to the fact that he flies 3 meters airborne.
I would prefer to keep the ironic tone.

Reply:
"Πουλί ήτανε;" is a legitimate Greek football expression for faking it or "taking a dive" (you hear football fans use it ubiquitously). Now that I mention it, I would prefer "taking a dive" here. 

"Being a bird" makes no sense to an English reader, he will not understand what you are talking about.